src/pages/groupComparison/GroupComparisonMutationMapperWrapper.tsx

```diff
@@ -149,20 +149,25 @@ export default class GroupComparisonMutationMapperWrapper extends React.Componen
             this.props.store.coverageInformation,
             this.props.store.groupToProfiledPatients,
             this.props.store.profiledPatientCounts,
+            this.props.store.genes,
+            this.props.store.genesSortedByMutationFrequency,
         ],
         render: () => {
-            const mutationMapperStore = this.mutationMapperToolStore.getMutationMapperStore(
-                this.props.store.activeMutationMapperGene!.hugoGeneSymbol
-            );
-            if (mutationMapperStore) {
+            let mutationMapperStore;
+            if (
```

Comment:
i think what it should be is 
```
render:()=> {
   if (this.props.store.activeMutationMapperGene.hugoGeneSymbol){
      const mutationMapperStore = ...
      return ()
   }

}
```

Reply:
mutationMapperStore can be undefined even if the hugoGeneSymbol is defined, so this would break. you still have to check if mutationMapperStore is defined or not